soc/nxp/imxrt/imxrt118x/soc.c

```diff
@@ -85,6 +89,17 @@ __attribute__((weak)) void board_flexspi_clock_safe_config(void)
 {
 }
 
+#if CONFIG_USB_DC_NXP_EHCI
+/* USB PHY condfiguration */
+#define BOARD_USB_PHY_D_CAL     (0x07U)
+#define BOARD_USB_PHY_TXCAL45DP (0x06U)
+#define BOARD_USB_PHY_TXCAL45DM (0x06U)
+
+usb_phy_config_struct_t usbPhyConfig = {
+	BOARD_USB_PHY_D_CAL, BOARD_USB_PHY_TXCAL45DP, BOARD_USB_PHY_TXCAL45DM,
+};
+#endif
```

Comment:
Why do you need these changes when there are devicetree properties that describe the same thing?

Reply:
These changes were added to support the old USB driver. I have deleted this code as its not needed for the USB_Next driver which as you rightly pointed out uses DT properties.